app/mailers/notification_mailer.rb

```diff
@@ -1,7 +1,7 @@
 class NotificationMailer < ApplicationMailer
-   default from: ENV["DEFAULT_FROM_ADDRESS"]
+  default from: ENV['DEFAULT_FROM_ADDRESS']
 
-   ALLY_NOTIFY_TYPES = ["new_ally_request", "accepted_ally_request"]
+  ALLY_NOTIFY_TYPES = %w(new_ally_request accepted_ally_request).freeze
```

Comment:
The indentation was off on these lines and it was causing Code Climate to report that the function I introduced was misaligned.  Since I touched these I had to make all of these changes to appease Rubocop.  😭